packages/react-notifications/src/components/notification-kinds/api-error-message/api-error-message.tsx

```diff
@@ -141,10 +141,16 @@ function getSpecialFormattedMessageByErrorCode(
   if (extensionErrorCode === 'insufficient_scope')
     return intl.formatMessage(messages.Forbidden);
 
-  if (extensionErrorCode === 'DuplicateField' && error.field === 'slug')
-    return intl.formatMessage(messages.DuplicateSlug, {
-      slugValue: error.duplicateValue,
-    });
+  if (extensionErrorCode === 'DuplicateField') {
+    if (error.field === 'slug')
+      return intl.formatMessage(messages.DuplicateSlug, {
+        slugValue: error.duplicateValue,
+      });
+    else
+      return intl.formatMessage(messages.DuplicateField, {
+        field: error.field,
+      });
```

Comment:
**nit** I see we are not consistent here about it, but I would love for use to use curly braces in conditional statements ❤️